configurations/default/env.yml.tmp

```diff
@@ -12,8 +12,8 @@ DISABLE_AUTH: false
 OSM_VEX: http://localhost:1000
 SPARKPOST_KEY: your-sparkpost-key
 SPARKPOST_EMAIL: email@example.com
-GTFS_DATABASE_URL: jdbc:postgresql://localhost/catalogue
+GTFS_DATABASE_URL: jdbc:postgresql://localhost/catalogue # If running via docker, this is jdbc:postgresql://postgres/dmtest
 # GTFS_DATABASE_USER:
 # GTFS_DATABASE_PASSWORD:
-#MONGO_HOST: mongo-host:27017
+#MONGO_HOST: mongo-host:27017 # If running via docker, this is mongo:27017
```

Comment:
Yes "mongo" is defined in the docker-compose file so it's fixed. However since this file is shared between docker and non-docker instances I've left the docker default as a comment.